node/modules/peering/peering.go

```diff
@@ -0,0 +1,45 @@
+package peering
+
+import (
+	"github.com/ipfs/go-ipfs/peering"
+	"github.com/libp2p/go-libp2p-core/host"
+	"github.com/libp2p/go-libp2p-core/peer"
+)
+
+type PeeringPeer struct {
+	ID        string   `json:"ID"`
+	Addrs     []string `json:"Addrs"`
+	Connected bool     `json:"Connected,omitempty"`
+}
+
+type EstuaryPeeringService struct {
```

Comment:
why not use `peering.PeeringService` directly? why wrap it in `EstuaryPeeringService` ?

Reply:
nevermind, this wrapper will be good for testing

Reply:
We should always create a wrapper if we want to use an existing functionality so we can extend or add more processing to it. If we decide to log all peers in a DB or make any events, we can just extend the wrapper.